test/test.js

```diff
@@ -0,0 +1,6 @@
+const server = require('./server.js');
```

Comment:
1. should use DELETE response to test these cases
2. please write your strategies into readme.md